python/shotgun_model/shotgunmodel.py

```diff
@@ -90,7 +90,7 @@ class ShotgunModel(QtGui.QStandardItemModel):
     FILE_VERSION = 21
 
 
-    def __init__(self, parent, download_thumbs=True, schema_generation=0):
+    def __init__(self, parent, download_thumbs=True, schema_generation=0, bg_thumbs=False):
```

Comment:
I feel like the name bg_thumbs is not clear enough. It sounds confusing when considering the the worker thread retrieves thumbnails in the background. Maybe bg_load_thumbs (or anything that implies that it is being not only downloaded but also read from disk by the background thread) would be clearer?